tsplib95/transformers.py

```diff
@@ -244,14 +244,6 @@ def pack(self, items):
     def unpack(self, container):
         return list(container)
 
-class TupleT(ContainerT):
-    """Transformer for tuples"""
-
-    def pack(self, items):
-        return tuple(items)
-
-    def unpack(self, container):
-        return tuple(container)
```

Comment:
I can't remember the reason why I included this class, but it has something to do with this PR: https://github.com/rhgrant10/tsplib95/pull/19